Websites/webkit.org/wp-content/themes/webkit/widgets/page.php

```diff
@@ -11,7 +11,7 @@
 class WebKitPageTileWidget extends WebKitPostTileWidget {
 
     public function __construct() {
-        parent::WP_Widget(false,
+        parent::__construct(false,
```

Comment:
My understanding is this will invoke constructor of WebKitPostTileWidget, which does not take any argument in its constructor. In this case, arguments will be ignored.